.github/workflows/sdk-javascript-release.yml

```diff
@@ -2,7 +2,8 @@ name: SDK JavaScript - Release
 on:
   push:
     paths:
-      - packages/sdks/javascript/**
+      - packages/sdks/javascript/dist
```

Comment:
Why `dist`?
This folder is never committed to the repo, so I don't see how it could trigger a release. 🤔 
Shouldn't we check for changed in `src` folder instead?